src/rt/sections_elf_shared.d

```diff
@@ -59,6 +59,12 @@ import rt.minfo;
 import rt.util.container.array;
 import rt.util.container.hashtab;
 
+void safeAssert(bool condition, scope string msg, scope string filename = __FILE__, size_t line = __LINE__) @nogc nothrow @safe
```

Comment:
This would be a sensible place to include your comment
>Normal assertions (throwing AssertErrors) aren't really usable as the GC may not be initialized (yet or anymore).